### ISSUE TITLE
Duplicate functions `the_excerpt_max_charlength`

### ISSUE THREAD

**Issue by matt-bernhardt** [MITLibraries/MITLibraries-news#85]
The function `the_excerpt_max_charlength` is defined twice, in page-last-year.php and archive-bibliotech.php. This is problematic for several reasons, particularly when there's also an `excerpt` function defined in functions.php

**Comment by PBruk** [comment]
Agreed, function should be moved/removed to `functions.php` and am pretty sure `page-last-year.php` isn't even being used?
